### DISCUSSION TITLE
Rainbow App Listed Tsuru on Base as a Free mint with gas cost of 0.02 cents. Was charged 0.0012 ETH

### DISCUSSION THREAD

**Discussion by DayOfShah** [rainbow-me/rainbowkit#1951] [Feedback]
Instead I was charged 0.0012 ETH, without any permission or consent. The actual cost may be insignificant, but misleading your community is unacceptable. Please rectify this matter

**Comment by DayOfShah** [comment]
Thank you for your message,
The false advertising was only present on Rainbow's platform. So why would
I speak to someone from Ethereum?

**Comment by DayOfShah** [comment]
Following up here

**Comment by DayOfShah** [comment]
Following up here again. Why did Rainbow tell me to contact Ethereum
regarding Rainbow's false advertising?

Please return my funds, or I will escalate this accordingly. Defrauding US
citizens is a crime. Has Rainbow wallet obtained the appropriate
licenses to offer its services in America?

**Comment by 0xmkh** [comment]
@DayOfShah Let's chat in this discussion https://github.com/rainbow-me/rainbowkit/discussions/1960

**Comment by DayOfShah** [comment]
There's nothing more to discuss. Rainbow falsely advertised a free mint
with a gas fee of $0.02 (2 cents), and then charged me 0.0012 ETH. I minted
this twice consecutively and during neither mint did this fee appear. It
was only after minting that I realized what had occurred. Then, upon being
confronted with this information, Rainbow insultingly attempted to mislead
me into believing this issue would be resolved through discourse with the
Ethereum Foundation. Rainbow should consider itself fortunate that I am
willing to drop the issue here. Return the funds Rainbow fraudulently
charged me immediately.

**Reply by 0xmkh** [reply]
Apologies for the inconvenience. The team is looking at this right now, please bear with us 🙏

**Comment by DayOfShah** [comment]
This is taking far too long. If this is an attempt to delay resolving the
matter in hopes of it being forgotten, I am informing you now that will
backfire. What exactly is the team “looking at”? Rainbow falsely advertised
a free mint; deceptively withdrew cryptocurrency without authorization; and
when confronted with this information, misleadingly advised me to contact
the Ethereum Foundation to resolve this matter. This was insulting, and
unethical.

If my property is not returned by 5pm on Monday May 6, 2024, I will
escalate and report this matter accordingly.
